<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>160</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="id"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="sex"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phonum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="birth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="idnum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="oriplace"/>
    </item>
    <item>
     <widget class="QLineEdit" name="remarks"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>111</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="m_id">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_name">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_sex">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_phnum">
      <property name="text">
       <string>联系方式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_birth">
      <property name="text">
       <string>出生日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_idNum">
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_nation">
      <property name="text">
       <string>民族</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_oriplace">
      <property name="text">
       <string>籍贯</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_remarks">
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="input">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确认输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ret">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
